.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
please delete this folder. commit and git push.  
when you use live server, make sure you only have one instance of live server running at a time, then it wont create a new port number. you only have to click Go Live once... once its on it will stay running for duration of vscode session.